code/modules/atmospherics/machinery/pipes/layeredmanifold4w.dm

```diff
@@ -0,0 +1,315 @@
+/* Four-way manifolds */
+
+/*===== Haute tayau *kisses fingers* ===== */
+//Supply
+
+/obj/machinery/atmospherics/pipe/manifold4w/supply/visible/bottom
+	level = PIPE_VISIBLE_LEVEL
```

Comment:
why?

Reply:
Rushed - forgot to remove unnecessary defines before I left for the day.

Reply:
alright - clean up these bits and bobs and 👍